### ISSUE TITLE
presenter mode not visible

### ISSUE THREAD

**Issue by QHose** [hakimel/reveal.js#1996]
hi,

I use reveal in my meteor app (node) and it works perfectly, only when I press S for presenter mode, nothing happens... any idea what could be issue?

thanks!

**Comment by rugk** [comment]
A popup blocker?

**Comment by QHose** [comment]
No, that is not the issue... it must be something very simple... :(

Verstuurd vanaf mijn iPhone

Op 3 nov. 2017 om 17:35 heeft rugk <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


A popup blocker?
